tests/test_Source/test_supernovae_pop.py

```diff
@@ -1,34 +1,47 @@
 from slsim.Sources.Supernovae.supernovae_pop import (
     calculate_star_formation_rate,
-    t_d_power,
+    delay_time_distribution,
 )
 from slsim.Sources.Supernovae.supernovae_pop import SNIaRate
 from astropy.cosmology import FlatLambdaCDM
 import numpy as np
 import numpy.testing as npt
 
 cosmo = FlatLambdaCDM(70, 0.3)
```

Comment:
can you make the tests go in it's now class for tests of class methods? See other tests that do that